agent/ecscni/plugin_test.go

```diff
@@ -259,3 +263,32 @@ func TestCNIPluginVersion(t *testing.T) {
 		})
 	}
 }
+
+// Returns the version in CNI plugin VERSION file as a string
+func getCNIVersionString(t *testing.T) string {
+	// ../../amazon-ecs-cni-plugins/VERSION
+	var versionFilePath = filepath.Clean(filepath.Join("..","..","amazon-ecs-cni-plugins","VERSION"))
+	fmt.Println(versionFilePath)
+	versionStr, err := ioutil.ReadFile(versionFilePath)
+	assert.NoError(t, err, "Error reading the CNI plugin version file")
+	return strings.TrimSpace(string(versionStr))
+}
+
+// Asserts that CNI plugin version matches the expected version
+func TestCNIPluginVersionNumber(t *testing.T) {
+	var versionStr = getCNIVersionString(t)
+	assert.Equal(t, currentCNIVersion, versionStr)
+}
+
+// Asserts that CNI plugin version is upgraded when new commits are made to CNI plugin submodule
+func TestCNIPluginVersionUpgrade(t *testing.T) {
+	var versionStr = getCNIVersionString(t)
+	cmd := exec.Command("git", "submodule")
+	versionInfo, err := cmd.Output()
+	versionInfoStr := string(versionInfo)
+	assert.NoError(t, err, "Error running the command: git submodule")
```

Comment:
should this be right after cmd.Output()? otherwise when err != nil, versionInfo = nil then string(nil) will fail